loguru/_handler.py

```diff
@@ -297,6 +317,7 @@ def _queued_writer(self):
     def __getstate__(self):
         state = self.__dict__.copy()
         state["_lock"] = None
+        state["_lock_acquired"] = None
```

Comment:
Very good catch.

Reply:
It was well was covered in tests :muscle: